uber/site_sections/registration.py

```diff
@@ -71,6 +79,9 @@ def form(self, session, message='', return_to='', omit_badge='', **params):
             attendee.group_id = params['group_opt'] or None
             if AT_THE_CON and omit_badge:
                 attendee.badge_num = 0
```

Comment:
I moved the check to ``model_checks.py`` since that seemed like a better place for it.